src/skill.rs

```diff
@@ -137,3 +137,28 @@ pub const SURVIVAL: SkillId = SkillId(16);
 
 /// Number of core skills.
 pub const RESERVED_SKILLS: u8 = 17;
+
+/// Returns a string representation of an `SkillId`.\
+/// The string is accurate only when using the skills defined by the SRD.
+pub fn srd_skill_string(id: SkillId) -> String {
+    match id.0 {
+        0 => "acrobatics".to_string(),
+        1 => "animal handling".to_string(),
+        2 => "arcana".to_string(),
+        3 => "athletics".to_string(),
+        4 => "deception".to_string(),
+        5 => "history".to_string(),
+        6 => "insight".to_string(),
+        7 => "intimidation".to_string(),
```

Comment:
what about `investigation`?

Reply:
oh no, I forgot it completely!

good catch!